Proposed cointype. (#4)
chaintope/tips: tip-hdwallet-cointype.md

```
TIP: HDWallet-cointype
Type: Application
Title: HDWallet cointype
Status: Draft
Author: Yukishige Nakajo <nakajo@chaintope.com>
Created: 2023-08-09
License: CC0
```

## Abstract

Define cointype of BIP-0044 for Tapyrus. I suppose to '2377' as cointype.

## Copyright

This TIP is licensed under the CC0 license.

## Motivation

> Now, Tapyrus is not defined cointype of BIP-0044. So, It is confuse what number set to cointype when implement HDWallet for tapyrus.

Reviewer: ```suggestion
Now, there is no cointype defined for Tapyrus in BIP-0044. So, It is confusing what number is set to cointype when implementing HDWallet for tapyrus.
```

> I suggest to propose to cointype for Tapyrus. but, We need consensus in this community before push to [SLIP-0044](https://github.com/satoshilabs/slips/blob/master/slip-0044.md)

Reviewer: ```suggestion
I propose to cointype for Tapyrus. But we need consensus in this community before pushing to [SLIP-0044](https://github.com/satoshilabs/slips/blob/master/slip-0044.md).
```
